00-HTML-CSS-basics/Punto 1/1.1/lista.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>My todo list</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+	<header>My todo list</header>
+	<ul>
+		<li>Bake a cake</li>
```

Comment:
Standarize your indentation, you can configure your favorite editor with 2 or 4 spaces for each 'tab'. Use this convention for html, css and js.